opta/core/terraform2.py

```diff
@@ -0,0 +1,195 @@
+import os
+from shutil import copyfile
+from subprocess import DEVNULL, PIPE  # nosec
+from typing import Any, Dict, List, Optional
+
+from opta.core.terraform import Terraform as LegacyTerraform
+from opta.exceptions import UserErrors
+from opta.layer2 import Layer
+from opta.nice_subprocess import nice_run
+from opta.utils import json
+
+
+class Terraform:
```

Comment:
should we start putting the terraform code in a separate package and break into smaller classes?
It could be easier to test
ex: 
```
core.terraform.Terraform
core.terraform.TerraformFile
```

Reply:
Separated these into their own files